pages/testCases/kundur_two_area_system/index.md

```diff
@@ -0,0 +1,475 @@
+| bus05    | $$1.006 \angle 13.737°$$  | \-                             |
+| bus06    | $$0.978 \angle 3.651°$$   | \-                             |
+| bus07    | $$0.961 \angle -4.759°$$  | \-                             |
+| bus08    | $$0.949 \angle -18.633°$$ | \-                             |
+| bus09    | $$0.971 \angle -32.234°$$ | \-                             |
+| bus10    | $$0.983 \angle -23.819°$$ | \-                             |
+| bus11    | $$1.008 \angle -13.511°$$ | \-                             |
+
+<div id="tbl-loadflowS">
+
+Table 10: Loadflow generator powers of Dynawo implementation
+</div>
+
+| Generator name | Complex power in MVA            | Kundur reference power in MVA |
+| :------------- | :------------------------------ | :---------------------------- |
+| G1             | $$700.105 + \mathrm{j}185.067$$ | $$700 + \mathrm{j}185$$       |
+| G2             | $$700.000 + \mathrm{j}234.676$$ | $$700 + \mathrm{j}235$$       |
+| G3             | $$719.000 + \mathrm{j}175.986$$ | $$719 + \mathrm{j}176$$       |
+| G4             | $$700.000 + \mathrm{j}202.071$$ | $$700 + \mathrm{j}202$$       |
+
+
+- The active power exchange from Area 1 to Area 2 (see
+  <a href="#fig-two-area-system" class="quarto-xref">Figure 1</a>) is
+  400.431 MW in the Dyna$$\omega$$o implementation and 400 MW in the
+  Kundur reference.
+
+### Example a
+
+*Example a* examines small-signal performance of the system with
+constant excitation.
+
+- All four generators are on manual excitation (field voltage
+  $$E_\mathrm{fd} = \mathrm{const.}$$)
+- Constant current characteristics of active load components
+- Constant impedance characteristics of reactive load components
+
+<a href="#fig-example-a-eigenvalues" class="quarto-xref">Figure 2</a>
+shows the result of an eigenvalue analysis of this system, compared to
+the reference values from [1]. The remaining differences probalby
+originate from differing model implementations of the synchronous
+machines in Dyna$$\omega$$ compared to the ones used by Kundur.
+
+
+![](images/20240814123344.png)
+
+<div id="fig-example-a-eigenvalues">
+
+Figure 2: Eigenvalues of the system with constant excitation voltage
+
+</div>
+
+### Example b
+
+> [!NOTE]
+>
+> To obtain a steady state initial condition, the exact values for the
+> generators active and reactive power and voltage angles from load flow
+> (<a href="#sec-loadflow" class="quarto-xref">Section 6.1</a>) need to
+> be used as initial values in the dynamic simulation.
+
+> [!NOTE]
+>
+> In the Dyna$$\omega$$o implementation, if the voltage transducer time
+> constant $$T_\mathrm{R}$$ = 0.01 s, then the voltage measurement first
+> order lag and the field flux linkage $$\lambda_\mathrm{f}$$ oscillate
+> against each other with high amplitude. This is not represented in the
+> reference results of [2]. To avoid this behaviour, set
+> $$T_\mathrm{R} = 10^{-6}$$ s. This has been done for the Dyna$$\omega$$o
```

Comment:
This really requires some investigation. In practice you say that the filter/transducer was removed from the model (which 
 model by the way? ST1C?)